docs/documentation/docs-guidelines.md

```diff
@@ -0,0 +1,43 @@
+# Docs Guidelines
+
+This guidelines direct authors in creating, naming and writing files providing
+information such as guidelines, how-tos, tutorials, examples or abouts.
+
+
+## Location
+When such new information is to be added, the following points need to be considered
+concerning the location of this information:
+
+* If information that applies to exactly one directory of the ILIAS
+repo is provided then a file named README.md MUST be created in this exact directory and
+the new information MUST be added there.
+* Each folder except docs and it's descendants MUST only contain at most one md file,
+named README.md
+* Information of general nature or not concerning strictly one directory MUST be placed
+ in the docs folder or even better in one of the following sub directories of docs if
+ applicable:
+  * documentation: For examples, tutorials, explanations and guidelines covering
+  cross-sectional matters. Mostly for contributors.
+  * configuration: For How-Tos, FAQs installation instructions or similar. Mostly for
+ administrators.
+* Any information essential for either running ILIAS or contributing to ILIAS SHOULD at
+ least be linked in the global README.md file placed in the main directory of the ILIAS
+  repo.
+
+## Naming
+* Files providing information for one exact folder MUST be named README.md.
+* Only the following files MUST be written in uppercase:
+  * README.md
+  * LICENSE.md
+* All other files MUST be written in lowercase.
+* Names containing more than one word MUST use hyphens to separate the words (such as
+docs-guidelines).
+* The file extension for all markdown files MUST be '.md'
+
+## Content
+* Markdown SHOULD be used for providing information such as described above.
```

Comment:
I second that. Markdown is easy readable as plain text, integrates well with Github and can be transformed into other formats without hassle.